docs/en/ingest-management/elastic-agent/install-agent-account-requirements.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[install-agent-account-requirements]]
+= {agent} account and permissions requirements
+
+There are a number of account and permissions requirements that should be met before running {agent} on your system.
+
+* <<elastic-agent-account-requirements-linux>>
+* <<elastic-agent-account-requirements-windows>>
+
+[discrete]
+[[elastic-agent-account-requirements-linux]]
+== Linux and Unix requirements
+
+Currently, on Linux and Unix systems {agent} needs to run as a super user with `sudo` privileges. To create a new user with `sudo` privileges:
+
+. Create a user for your {agent}.
++
+[source,sh]
+----
+useradd elastic
+----
+
+. Add the new user to the `sudo` group.
++
+[source,sh]
+----
+usermod -aG sudo elastic
+----
+
+. Proceed with the instructions to <<install-fleet-managed-elastic-agent>> or <<install-standalone-elastic-agent>>.
+
+As well, {agent} components (Beats) each use a unique account with specific controlled `sudo` privileges, as shown in the following table.
+
+.Linux and Unix requirements for agent components
+|===
+|Component |Function |Requirements
+
+|{auditbeat}
+|Monitors the activities of users and processes on your systems.
+a|* `sudo` privilege to run the `auditd` command.
+* Permissions to read and audit directories, such as `/user/bin`, `user/sbin`, `/bin`, `/etc`.
+* Permissions to read logs in `/var/log`.
+* Permissions to read and write logs cached in `/var/log/Elastic/.
+
+|{heartbeat}
+|Monitors the status of your services.
+a|* `sudo` privilege to run the `auditd` command.
+* `sudo` privilege to run commands to check disk IO, network IO, memory, process, and CPU level statistics.
+* Permissions to read and audit directories, such as `/user/bin`, `user/sbin`, `/bin`, `/etc`.
+* Permissions to read logs in `/var/log`.
+* Permissions to read and write logs in cache to `/var/log/Elastic/.
+
+|{filebeat}
+|Forwards and centralizes your log data.
+a|* Permissions to read logs in `/var/log`.
+* Permissions to read and write logs in cache to `/var/log/Elastic/.
+
+|{metricbeat}
+|Collect metrics from operating systems and services running on your servers.
+a|* `sudo` privilege to run commands to check disk IO, network IO, memory, process, and CPU level statistics.
+* Permissions to read logs in `/var/log`.
```

Comment:
I don't think Metricbeat reads logs in general. This line is repeated for each Beat, but I'm not sure where it came from.